invenio_communities/assets/semantic-ui/js/invenio_communities/community/communitiesCarousel/CarouselItem.js

```diff
@@ -70,11 +90,13 @@ CarouselItem.propTypes = {
   defaultLogo: PropTypes.string.isRequired,
   className: PropTypes.string,
   showUploadBtn: PropTypes.bool,
+  canSubmitRecord: PropTypes.bool,
```

Comment:
```suggestion
  canSubmitRecord: PropTypes.bool.isRequired,
```

you always pass the property so having a default is redundant